apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notices.js

```diff
@@ -147,7 +147,8 @@ function GlobalStylesEditNotice() {
 				onClick: upgradePlan,
 				variant: 'primary',
 				noDefaultClasses: true,
-				className: 'wpcom-global-styles-action-has-icon wpcom-global-styles-action-is-external',
+				className:
+					'wpcom-global-styles-action-is-upgrade wpcom-global-styles-action-has-icon wpcom-global-styles-action-is-external',
```

Comment:
I think it's easier to get that there are three different classnames like this:

```suggestion
				className: classNames(
					'wpcom-global-styles-action-is-upgrade',
					'wpcom-global-styles-action-has-icon',
					'wpcom-global-styles-action-is-external'
				),
```

Reply:
Makes sense, this is done :)

Reply:
thanks!